RPGManager_development/src/rpg_database/character_sheet/LanguagesSet.java

```diff
@@ -0,0 +1,115 @@
+package rpg_database.character_sheet;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.Set;
+
+import rpg_database.character_sheet.interfaces.CustomSetter;
+
+public class LanguagesSet implements Set<Languages>, CustomSetter<LanguagesSet>{
+	
+	private final HashSet<Languages> languagesSet;
+	
+	public LanguagesSet(Languages...languages){
+		languagesSet = new HashSet<Languages>();
+		languagesSet.addAll(Arrays.asList(languages));
+	}
+	
+	@Override
+	public Class<LanguagesSet> getImplementingClass() {
+		return LanguagesSet.class;
+	}
+	
+	@Override
+	public void setSelfInSheet(CharacterSheet characterSheet) {
+		LanguagesSet characterSheetLanguages = (LanguagesSet) characterSheet.characterData.get(Fields.LANGUAGES);

```

Comment:
Ne haragudj de itt nem értem hogy mi történik. Az a languages változó ott mire kell? Meg a for ciklus? Azt írtad hogy megcsinálod ezt a függvényt úgy hogy a setData a ezzel a (this) LanguagesSet-tel felülírja az előzőt. Ehhez képest látom hogy még mindig kikéred az előzőt és ahhoz addolgatsz.